draft-ietf-tls-tls13.md

```diff
@@ -1601,39 +1610,34 @@ CertificateRequest
   [[OPEN ISSUE: See https://github.com/tlswg/tls13-spec/issues/184]].
   [{{certificate-request}}]
 
-CertificateVerify
-: a signature over the entire handshake using the public key
-  in the Certificate message. This message will be omitted if the
-  server is not authenticating via a certificate. [{{server-certificate-verify}}]
-
-Finished
-: a MAC over the entire handshake computed using the Static Secret.
-  This message provides key confirmation and
-  In some modes (see {{zero-rtt-exchange}}) it also authenticates the handshake using the
-  the Static Secret. [{{server-finished}}]
-{:br }
 
-Upon receiving the server's messages, the client responds with his final
-flight of messages:
+Finally, the client and server exchange Authentication messages. TLS
+uses the same set of messages every time that authentication is needed.
+Specifically:
 
 Certificate
-: the client's certificate. This message will be omitted if the
-  client is not authenticating via a certificates. [{{client-certificate}}]
+: the certificate of the endpoint. This message is omitted if certificate
+  authentication is not being used. [{{certificate}}]
 
 CertificateVerify
-: a signature over the entire handshake using the private key corresponding
-  to the public key in the Certificate message. This message will be omitted if the
-  client is not authenticating via a certificate. [{{client-certificate-verify}}]
+: a signature over the entire handshake using the public key
+  in the Certificate message. This message will be omitted if the
+  server is not authenticating via a certificate. [{{certificate-verify}}]
 
 Finished
-: a MAC over the entire handshake computed using the Static Secret
-  and providing key confirmation. [{{server-finished}}]
+: a MAC over the entire handshake. This message provides key confirmation, binds the endpoint's identity
+  to the exchanged keys, and in some modes (see {{zero-rtt-exchange}}) 
```

Comment:
Do you mean to reference the 0RTT handshake, or do you mean to refer to PSK/resumption?